packages/tests/src/nwaku.ts

```diff
@@ -117,7 +117,7 @@ export class Nwaku {
     }
 
     return {
-      payload: Buffer.from(message.payload).toString("base64"),
```

Comment:
`base64` is the correct format.

See https://github.com/waku-org/nwaku/issues/1616